### PR TITLE
Switch Dask project to HSF-India

### PR DESCRIPTION
**Pull request by jpivarski**
@oshadura and @ianna; I've switched the categories, but not the text of the abstract.

### CODE REVIEW

File: projects/dask-in-coffea-casa.yml [hunk 1]
```diff
@@ -13,13 +13,13 @@ skillset:
 status:
   - Available
 project:
-  - IRIS-HEP
+  - compiler research
```
Comment (alexander-held) [inline projects/dask-in-coffea-casa.yml:16]: I believe this should be some HSF India value instead? I cannot find the syntax for that though in the `project_metadata.yml` or any other project.

Comment (jpivarski) [reply projects/dask-in-coffea-casa.yml:16]: I saw something yesterday that convinced me that "program: HSF-India fellow" goes under "project: compiler research", but now I don't remember what it was.

Looking at all of the other projects, it seems that this is the first one that would be put under the HSF-India Traineeship program. Maybe the project_metadata.yml is just missing this project?

@davidlange6, is this correct?

**Review by ianna — APPROVED** [review]
@jpivarski - thanks!

**Comment by jpivarski** [comment]
@davidlange6, since the Trainee and Fellowship projects have just been announced, it would be good to merge this PR to avoid confusion. Thanks!

**Pull request merged**